content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -548,52 +548,8 @@ parameters:
 
 ### Quobyte
 
```

Comment:
If deprecating this, try:
```suggestion

{{< feature-state for_k8s_version="v1.22" state="deprecated" >}}

The Quobyte storage plugin is deprecated.
```

and then _leave in_ the docs about how to use it, unless there's a strong reason to remove them early.

When we remove in-tree Quobyte support, we can delete this section entirely.

Reply:
Sounds good.